keyboards/kbdfans/kbd75/keymaps/tucznak/keymap.c

```diff
@@ -0,0 +1,95 @@
+#include QMK_KEYBOARD_H
+
+#define _______ KC_TRNS
```

Comment:
QMK defines aliases for these keycodes automatically. If you use the already-defined aliases, Lines 3 and 4 can be deleted.

https://github.com/qmk/qmk_firmware/blob/a120a000d481ed14a87988383be46c5a76604bb6/quantum/quantum_keycodes.h#L29-L31